scripts/prepare_chunks.sh

```diff
@@ -66,22 +66,34 @@ mkdir -p $OUTDIR
 
 vg_chunk_params=(-x $GRAPH_FILE -g -c 20 -p $REGION -T -b $OUTDIR/chunk -E $OUTDIR/regions.tsv)
 
+
 # construct track JSON for graph file
-jq -n --arg trackFile "${GRAPH_FILE}" --arg trackType "graph" --argjson trackColorSettings '{"mainPalette": "plainColors", "auxPalette": "greys"}' '$ARGS.named' >> $OUTDIR/tracks.json
+GRAPH_FILE_PATH=$(realpath --relative-to ../ $GRAPH_FILE)
```

Comment:
Isn't `../` here just the directory above the directory the user is standing in when they run the script? If they do something like:
```
TUBE_MAP_DIR="$(pwd)"
cd exampleData
mkdir project1
cd project1
mkdir dataset1
cd dataset1
wget https://random-graph.info/graph.xg
${TUBE_MAP_DIR}/scripts/prepare_chunk.sh -x graph.xg -r chr1:1000-2000 -o region1 >graph.bed
```

Then I don't think this would work.

We might want to get the script's own path, and use that to get the root directory of the tube map repo, and make the path relative to that? Or we could make the path relative to wherever `tracks.json` is put, and make the server resolve the path relative to where `tracks.json` is.